integration_tests/gdk/components/test_integ_BuildCommand.py

```diff
@@ -26,8 +26,14 @@ def test_GIVEN_zip_build_system_WHEN_build_THEN_build_zip_artifacts(self):
         bc = BuildCommand({})
         bc.run()
         build_recipe_file = self.tmpdir.joinpath("greengrass-build/recipes/recipe.yaml").resolve()
+        included_file_path = "zip-build/" + self.tmpdir.name + "/src/test_do_want_this_file.txt"
+        excluded_file_path = "zip-build/" + self.tmpdir.name + "/test_dont_want_this_file.txt"
```

Comment:
use fstrings

Reply:
done in new commit

Reply:
Why should we use fstrings instead of concatenation @patrzhan?

Reply:
Both readability and so that we don't need to be worried about the type of the thing we are concatenating.